### PR TITLE
Add team introduction

### PR DESCRIPTION
**Pull request by palletprotocol**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -210,6 +210,102 @@
...
+            </li>
+            <li class="ul-team-li"><img src="img/team/AFH.png">
+            	<h4>Akiyoshi Fukumitsu</h4>
+                <h5>Founder & CEO of Hivelocity Inc.</h5>
+                <p class="exp">Mr. Fukumitsu is based in Tokyo, has over 20 years of web development, online marketing and business solutions extensive experience.
+</p>
+                
+            </li>
+            <li class="ul-team-li"><img src="img/team/kenneth.png">
+            	<h4>kenneth Chen</h4>
```
Comment (domen111) [inline index.html]: kenneth -> Kenneth

File: index.html [hunk 2]
```diff
@@ -210,6 +210,102 @@
...
+        <div class="title"><span>Team</span></div>
+    	<ul class="ul-team ul-en">
+        	<li class="ul-team-li"><img src="img/team/josh.png">
+            	<h4>Yuan Dao</h4>
+                <h5>Pallet thinker</h5>
+                <p class="exp">Founder of Token economy. Director of Z-Park Blockchain Industry Alliance. Promoter of Blockchain and “Tong Zheng“（Token）conception. Graduated from the Department of Electrical Engineering of Tsinghua University in 1991. Engaged in the Internet industry for more than 20 years. Leader of  the Internet infrastructure field.</p>
+            </li>
+            <li class="liao ul-team-li"><img src="img/team/liao.png">
+            	<h4>Dr.Block</h4>
+                <h5>Chief scientist of Pallet</h5>
```
Comment (domen111) [inline index.html]: scientist -> Scientist

**Comment by magrady24** [comment]
NEVER use jQuery for now.
Use &amp; for symbol "&" instead.
Need further reviews.

**Comment by DeemoHarlos** [comment]
Coding style severely unmatch.
Also, we use our own defined function instead of JQuery.
Pull request rejected.

**Comment by palletprotocol** [comment]
I have modified file index.html and deleted file jquery.min.js. Please review.

**Comment by magrady24** [comment]
There are some typo of the name of our team.
Need to fix.

**Comment by magrady24** [comment]
Please split the pull request into multiple parts, especially for the modifies of coding styles which has not been discussed and not incompatible.

**Comment by palletprotocol** [comment]
Now I close it. if it is needed later, it is OK to reopen it.

**Pull request closed without merging**